lib/types/application/ApplicationBuilder.js

```diff
@@ -4,6 +4,7 @@ const {getTask} = require("../../tasks/taskRepository");
 class ApplicationBuilder extends AbstractBuilder {
 	addStandardTasks({resourceCollections, project, log}) {
 		if (!project.metadata.namespace) {
+			// TODO 3.0: Throw here
```

Comment:
We missed that for 2.0, right?

Reply:
Yes, and now we can't change it without possibly breaking direct consumers 😕